LICENSE.md

```diff
@@ -47,11 +47,10 @@ http://hg.grauw.nl/grauw-lib
 
 https://github.com/cujojs/when
 
-> Open Source Initiative OSI - The MIT License
+>  Licensed under the MIT License at:
+>  http://www.opensource.org/licenses/mit-license.php
 >
-> http://www.opensource.org/licenses/mit-license.php
->
-> Copyright (c) 2011 Brian Cavalier
+>  MIT License (c) copyright B Cavalier &amp; J Hann
```

Comment:
yep! sorry for any confusion, guys. this is my first contribution ever. in
the universe.

On Wed, Dec 2, 2015 at 11:35 AM, Matthew Amato <notifications@github.com>
wrote:

> In LICENSE.md
> <https://github.com/AnalyticalGraphicsInc/cesium/pull/3235#discussion_r46463163>
> :
>
> >  >
> > -> http://www.opensource.org/licenses/mit-license.php
> > ->
> > -> Copyright (c) 2011 Brian Cavalier
> > +>  MIT License (c) copyright B Cavalier &amp; J Hann
>
> Ahh, that makes sense. Nice catch.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AnalyticalGraphicsInc/cesium/pull/3235/files#r46463163>
> .
>